firestore-send-email/functions/src/logs.ts

```diff
@@ -81,6 +81,14 @@ export function missingUids(uids: string[]) {
   );
 }
 
+export function noPartialAttachmentSupport() {
```

Comment:
Can this function take some additional args so that we can provide more context to the developer? That way, instead of `"partial attachments are not handled and will be ignored"`, we could do something like:

```js
logger.warn(`Ignoring partial attachment for templace "${docId}". The firestore-send-email extension does not support partial attachments.`)
```

Reply:
Let's follow up with this in another release